docs/cypress-testing-library/intro.md

```diff
@@ -53,12 +53,12 @@ To show some simple examples (from
 
 ```javascript
 cy.findAllByText('Jackie Chan').click()
-cy.queryByText('Button Text').should('exist')
-cy.queryByText('Non-existing Button Text').should('not.exist')
-cy.queryByLabelText('Label text', {timeout: 7000}).should('exist')
-cy.get('form').within(() => {
-  cy.findByText('Button Text').should('exist')
-})
+cy.findByText('Button Text').should('exist')
+cy.findByText('Non-existing Button Text').should('not.exist')
+cy.findByLabelText('Label text', {timeout: 7000}).should('exist')
+
+// findAllByText _inside_ a form element
+cy.get('form').findByText('Button Text').should('exist')
 cy.get('form').then(subject => {
   cy.findByText('Button Text', {container: subject}).should('exist')
 })
```

Comment:
What do people think about removing these lines? It is still supported, but the above example is much cleaner and more natural to other Cypress code.